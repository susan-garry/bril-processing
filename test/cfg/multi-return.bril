@foo: int {
    cond: int = const true;
    br cond .left .right;
  .left:
    a: int = const 1;
    ret a;
  .right:
    a: int = const 0;
    ret b;
}

@main {
  res: int = call @foo;
  print res;
}