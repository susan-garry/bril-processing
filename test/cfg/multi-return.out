func:  foo
  block0: ['left', 'right']
  left: []
  right: []
func:  main
  block0: []
